backend/src/cache/server/mutation.go

```diff
@@ -34,6 +34,8 @@ import (
 const (
 	KFPCacheEnabledLabelKey   string = "pipelines.kubeflow.org/cache_enabled"
 	KFPCacheEnabledLabelValue string = "true"
+	KFPCachedLabelKey         string = "pipelines.kubeflow.org/cached"
```

Comment:
What do you think about "pipelines.kubeflow.org/reused_from_cache"?

Reply:
Sure, sounds better!